scripts/index/publish_artifacts.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import os
+import boto3
+import json
```

Comment:
The import should happen before `import boto3` and also `import os`. Adding a blank line between the stdlib/3rdparty imports would be great.